independent-projects/qute/core/src/main/java/io/quarkus/qute/SetSectionHelper.java

```diff
@@ -68,7 +70,12 @@ public Scope initializeBlock(Scope previousScope, BlockInfo block) {
                 Scope newScope = new Scope(previousScope);
                 for (Entry<String, String> entry : block.getParameters().entrySet()) {
                     Expression expr = block.addExpression(entry.getKey(), entry.getValue());
-                    newScope.putBinding(entry.getKey(), expr.collectTypeInfo());
+                    //newScope.putBinding(entry.getKey(), expr.collectTypeInfo());
```

Comment:
Why the comment?

Reply:
Good catch. It should not be there at all...